apps/src/appMain.js

```diff
@@ -62,6 +62,13 @@ export default function (app, levels, options) {
 
   studioApp().configure(options);
 
+  if (options.level && options.level.shapeShift) {
+    const rotate = m => m[0].map((x, i) => m.map(x => x[m.length - i - 1]));
+    for (let i = 0; i < Math.random() * 4; i++) {
```

Comment:
Nit: This won't give you an even distribution of rotations, and you probably want it to do 0-3 rotations, rather than 1-4, so something like `for (let i = 1, max = Math.random() * 4; i < max; i++) { ... }`